hsctl

```diff
@@ -253,8 +253,8 @@ migrate_hs() {
 
 manage_py_hs() {
     echo "MANAGEPY:"
-    echo "  - docker exec -u hydro-service -ti hydroshare python manage.py ${@:2}"
-    docker exec -u hydro-service -ti hydroshare python manage.py ${@:2}
+    echo "  - docker exec -u hydro-service -ti hydroshare python -W ignore::DeprecationWarning:: manage.py ${@:2}"
+    docker exec -u hydro-service -ti hydroshare python -W ignore::DeprecationWarning:: manage.py ${@:2}
```

Comment:
@aphelionz I think it is certainly good to ignore deprecation warning when developing code since there are so many deprecation warnings which clutter output considerably. However, during django upgrade work, these deprecation warning are good indications on what to work on to be able to make our code base to work on newer versions. So if we apply this globally, I'd suggest to make it a switch so that it can be turned on or off with default being ignoring deprecation warnings. In terms of upgrade path, I think we need to first to address all these deprecation warnings, then run tests to make sure all tests pass, at which point, we should be confident that most of functionalities are migrated to newer versions successfully and we then can ask the team to do manual testing to catch those not covered by unit tests.

Reply:
Good thoughts @hyi. How about a --verbose or -v flag for gas to? Default off.

Reply:
@aphelionz Sounds good to me.